ccmlib/scylla_cluster.py

```diff
@@ -96,13 +96,13 @@ def start(self, no_wait=False, verbose=False, wait_for_binary_proto=False,
                                profile_options=profile_options)
                 # Let's ensure the nodes start at different times to avoid
                 # race conditions while creating system tables
-                time.sleep(1)
+                time.sleep(0.1)
                 started.append((node, p, mark))
 
         if no_wait and not verbose:
             # waiting 2 seconds to check for early errors and for the
             # pid to be set
-            time.sleep(2)
+            time.sleep(0.1)
```

Comment:
this is without a loop its changing the behavior - not sure it will not cause tests to fail on slow machines

Reply:
This  is nowait branch - I can just as well remove the sleep altogether

Reply:
AFAIK no_wait means you do not wait for cql port to be open ..., so you can't remove the sleep